src/modules/status.rs

```diff
@@ -226,6 +229,55 @@ fn status_signal_name(signal: SignalNumber) -> Option<&'static str> {
     }
 }
 
+#[cfg(not(windows))]
+fn status_ntstatus_name(exit_code: &str) -> Option<&str> {
```

Comment:
```suggestion
fn status_ntstatus_name(_exit_code: &str) -> Option<&str> {
```

Clippy is upset by the unused variable (I'm actually surprised it didn't fail the CI entirely, it's usually a lot more strict). Prepend `_` to the argument to suppress the warning.

Reply:
Ah, good catch, thanks. Fixed it.